notebook/notebook/notebook.py

```diff
@@ -66,6 +76,36 @@ def get_name(line):
         "working directory must contain a file called `notebook-worker-images' "
         "containing the name of the docker image to use for worker pods.") from e
 
+CREATED_STATE = 'created'
+DELETED_STATE = 'deleted'
+
+AUTH_GATEWAY = os.environ.get("AUTH_GATEWAY")
+DB = os.environ.get("MYSQL_DB")
+IP = os.environ.get("MYSQL_IP")
+PORT = int(os.environ.get("MYSQL_PORT"))
+USER = os.environ.get("MYSQL_USER")
+PASS = os.environ.get("MYSQL_PASS")
```

Comment:
If we ultimately use a sql db, we'll need to modify `deployment.yaml.in` to set these values.

Reply:
Sure. Could you tell me why you prefer make over reading a .env? What advantages does it have?

Reply:
I don't care if its in a file in the docker image or an environment variable in the deployment, but either the `deployment.yaml.in` needs to change or the `Dockerfile` needs to change to include these configuration parameters. Probably, since the password is sensitive, that will need to be loaded from a k8s secret specified in `deployment.yaml.in`.

Reply:
env vars in the deployment are somewhat nicer because they're immediately visible when you `kubectl describe POD_NAME`.

Reply:
Wouldn't you still need to os.environ.get even if they're available in the environment?